kedro/framework/cli/project.py

```diff
@@ -71,7 +71,15 @@ def project_group():  # pragma: no cover
 @forward_command(project_group, forward_help=True)
 @click.pass_obj  # this will pass the metadata as first argument
 def test(metadata: ProjectMetadata, args, **kwargs):  # pylint: disable=unused-argument
-    """Run the test suite."""
+    """Run the test suite. (DEPRECATED)"""
+
+    deprecation_message = (
```

Comment:
Nit: I'm overly picky, but it would be nice to be consistent on space between docstring and `deprecation_message = ...` and between `click.secho` and the remaining logic below, IMO. :) I checked the previously-deprecated stuff, and it's not consistent, so I don't have a recommendation one way or another--just generally prefer if it's all the same.

Reply:
I wish the linter would do this for me, but it didn't. I will make a change to the styling.

Reply:
I move all the deprecated message at the top of the function, and a newline between the `click.secho` and the actual logic.